lib/matplotlib/backend_bases.py

```diff
@@ -3182,14 +3182,62 @@ def press_pan(self, event):
         if (event.button not in [MouseButton.LEFT, MouseButton.RIGHT]
                 or event.x is None or event.y is None):
             return
-        axes = [a for a in self.canvas.figure.get_axes()
-                if a.in_axes(event) and a.get_navigate() and a.can_pan()]
-        if not axes:
+
+        # Get all axes and group them by their navigation-capture behaviour.
+        axes_capture = {True: {}, False: {}}
+        for a in self.canvas.figure.get_axes():
+            if a.in_axes(event) and a.get_navigate() and a.can_pan():
+                # By default, axes with an invisible patch forward events
+                # while axes with a visible patch capture events.
+                if a.get_capture_navigation_events() == "auto":
+                    if a.patch.get_visible():
+                        axes_capture[True].setdefault(
+                            a.get_zorder(), []).append(a)
+                    else:
+                        axes_capture[False].setdefault(
+                            a.get_zorder(), []).append(a)
+                else:
+                    axes_capture[a.get_capture_navigation_events()].setdefault(
```

Comment:
I only need to check for `"auto"` because in any other case the capturing behavior is given by the value of `a.get_capture_navigation_events()` (e.g. to provide the requested ability to manually override the capturing behavior)

... or do you mean an explicit test that the value must be one of [True, False, "auto"] ? (If yes, I'd rather put the check in `a.set_capture_navigation_events()` directly)

Reply:
I believe @jklymak is talking about unit tests, not the test condition for the `if`, because codecov says this line is untested.